### PR TITLE
cart added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+                    <div class="card-actions">
+                        <button class="card-action-btn cart-btn">
+                            Proceed to pay
+                        </button>
+                    </div>
+                </div>
+
+
+
```
Comment (gauravsinhaweb) [inline cart/cart.html:151]: remove extra spacing here.

**Review by gauravsinhaweb — APPROVED** [review]
The UI looks so good, (Just a suggestion: add some font style to the text).  Good to merge.

File: cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            <div class="nav-items flex-flow-row">
+                <div class="nav-item"><a><img src="../images/logo.png" height="40rem" width="100rem"></a></div>
+                <div class="cart-info">
+                        <h3>Bag ------- Address ------- Payment</h3>
+                  </div>
+                  <div class="nav-items-right flex-flow-row">
+                    <div class="nav-item"><a href="signup-login/signup.html"><i class="fa fa-user fa-lg" aria-hidden="true"></i></a></div>
+                    <div class="nav-item"><a><i class="fa fa-heart fa-lg" aria-hidden="true"></i></a></div>
+                    <div class="nav-item"><a><i class="fa fa-shopping-bag fa-lg" aria-hidden="true"></i></a></div>                     
+                  </div>
```
Comment (Ankur9669) [inline cart/cart.html:24]: fix: Your nav links are not working.

**Review by Ankur9669 — APPROVED** [review]
UI suggestions:
1. I see you are using white as a background color, which is harsh for the user eyes, so you can use shades of white.
2. To me font is not looking good, you can change the font. It will make your UI more appeasing.

File: cart/cart.html (hunk 2 above)
Comment (swstica) [reply cart/cart.html:24]: Made all the changes, please review it

**Pull request merged**